### DISCUSSION TITLE
Live chat - #weblate at libera.chat

### DISCUSSION THREAD

**Discussion by Svetlana-T** [WeblateOrg/weblate#6102] [General] [answered]
Hi, some questions are better resolved interactively, such as [this one](https://github.com/WeblateOrg/weblate/issues/3106#issuecomment-850128100). Does weblate have or wish to have an IRC channel, for example, at [OFTC](http://www.oftc.net) or [libera](http://libera.chat)? I would be happy to learn there, and to volunteer to help others, it would be a bit easier for some people than monitor issues here?

**Comment by aparcar** [comment]
Any update on this? Is there no public channel for Weblate?

**Reply by Svetlana-T** [reply]
Hi aparcar, no I haven't heard anything lately. 

I believe that if this were of interest, we would need someone who is one of the lead developers for weblate, preferably the one who owns the domain, to register the channel and appoint several channel admins.

**Comment by nijel** [comment, marked as answer]
We used to be on Gitter, but due to low interest, nobody is attending it regularly: https://gitter.im/WeblateOrg/Lobby

I've now registered `#weblate` at https://libera.chat/. You can join it using webchat at https://web.libera.chat/#weblate.

**Comment by Svetlana-T** [comment]
A note in reference to gitter. I had preferred IRC as it does not require guests to create an account before they can participate.

**Reply by nijel** [reply]
Okay, let's give it a try, there is now also `#weblate` on libera.chat.

**Comment by Svetlana-T** [comment]
thanks for that, i'm in!

where could it be documented?

**Reply by nijel** [reply]
Adding it to the README as you've done in https://github.com/WeblateOrg/weblate/pull/7163 is probably good starting point.

**Reply by nijel** [reply]
I've also pinned this discussion, upvoting https://github.com/WeblateOrg/weblate/discussions/6102#discussioncomment-2056637 to be the first displayed reply will probably make it easier to find ;-).
